.github/actions/spell-check/whitelist.txt

```diff
@@ -0,0 +1,3240 @@
+aaaaa
+aaaaaa
+aaaaaaaaaaaaaaaaaaa
+AAAAAABBBBBBCCC
+AAAAABBBBBBBCCC
+AAD
+ABANDONFONT
+abcd
+abcde
+abcdef
+ABCDEFG
+ABCDEFGH
+ABCDEFGHIJ
+abcdefghijk
+abcdefghijklmnop
+ABCDEFGHIJKLMNOPQRST
+abcdefghijklmnopqrstuvwxyz
+ABCG
+abe
+abec
+abf
+abi
+acb
+accd
+ACCEPTFILES
+ACCESSDENIED
+acec
+acf
+acidev
+ACIOSS
+ACover
+actctx
+ACTCTXW
+activatable
+ACTIVEBORDER
+ACTIVECAPTION
+ADDALIAS
+ADDREF
+addressof
+ADDSTRING
+ADDTOOL
+AEB
+aeed
+aef
+AEnd
+afae
+afceeeaa
+AFE
+AFEF
+AFew
+AFill
+AFX
+AHelper
+ahz
+AImpl
+AInplace
+akb
+ALIGNRIGHT
+alloc
+allocing
+alpc
+ALTERNATENAME
+ALTF
+ALTNUMPAD
+ALWAYSTIP
+amd
+ansd
+ansicode
+ansicpg
+ANSISYS
+ANSISYSRC
+ANSISYSSC
+antialias
+antialiasing
+ANull
+Anup
+anycpu
+AOn
+APARTMENTTHREADED
+APCs
+api
+APIENTRY
+apiset
+apos
+appconsult
+appdata
+APPICON
+appium
+applet
+appletname
+applicationmodel
+APPLMODAL
+APPLOGIC
+appmodel
+apps
+APPWINDOW
+appx
+appxbundle
+Appxmandifest
+appxmanifest
+appxsources
+APrep
+aps
+apsect
+APSTUDIO
+architected
+argb
+argc
+args
+argv
+arigatoo
+ARRAYSIZE
+ARROWKEYS
+ASBRST
+ASBSET
+ASDF
+asdfghjkl
+ASetting
+ASingle
+asm
+asmv
+asmx
+aspnet
+aspx
+astextplain
+AStomps
+ASYNCWINDOWPOS
+atch
+ATest
+atg
+ative
+ativecxapp
+atl
+atlmfc
```

Comment:
How did these get in here? Are you not excluding all included files? This feels like stuff that isn't in our code. (`ative`, `ativecxapp`...)

Reply:
Oh are these "NATIVE" and "NATIVECXAPP" with the leading character? Man it would be nice to be able to put comments in this file.

Reply:
https://github.com/microsoft/terminal/blob/be77b2a2e98420db9d275d68f35b55ac58bc93c0/src/host/ft_integrity/sources.dep#L5
Yes this is the `\n` stuff. We could add `.dep` to excludes

https://github.com/microsoft/terminal/blob/be77b2a2e98420db9d275d68f35b55ac58bc93c0/src/unit.tests.x64.runsettings#L86
Similarly, `.runsettings` could be added to excludes

I imagined comments would be hard.

What would be fairly doable today is to commit this file in stages w/ commit messages explaining each piece, then git blame / github blame could explain.

I could probably cheat and treat the file as `Token COMMENT` -- i.e. strip everything after the whitespace (or define a comment character, e.g. `#`). Talk this over w/ me, and I can put it into a 0.0.12. (0.0.11 fixes a minor bug in the whitelist suggestion comment).

Reply:
All `sources.dep` files would be excluded as artifacts of the Windows build system (as would all `*.pbxproj`, `sources`, `dirs`, and a few other things).

Yeah, I was thinking that you could just discard everything after the # at the end of a line. I like that idea.